integrationtest/src/main/java/software/amazon/timestream/integrationtest/Constants.java

```diff
@@ -26,8 +26,35 @@ final class Constants {
   static final String URL = "jdbc:timestream";
   static final String DATABASE_NAME = "JDBC_Integration07_Test_DB";
   static final String TABLE_NAME = "Integration_Test_Table_07";
+
+  static final String NO_DB_NO_TB_REGION = "ap-southeast-2";
+
+  static final String[] NO_DB_DATABASE_NAMES = new String[] {};
+
+  static final String[] NO_TB_TABLE_NAMES = new String[] {};
+
+  static final String ONE_DB_NO_TB_REGION = "ap-northeast-1";
+
+  static final String[] ONE_DB_DATABASE_NAMES = new String[] {
+          "JDBC_Inte.gration_Te.st_DB_01"
+  };
+
+  static final String[] ONE_DB_NO_TB_TABLE_NAMES = new String[] {};
+
+  static final String ONE_DB_ONE_TB_REGION = "eu-central-1";
+
+  static final String[] ONE_DB_ONE_TB_DATABASE_NAME = new String[] {
+          "JDBC_.IntegrationTestDB0088"
```

Comment:
I think there is no need to make the DB and table name different from others. The test logic is to test one db with one table could work fine. Not test the different names. But if you like it, you can keep it.

Reply:
Yea I just wanted to test different patterns

Reply:
It could be done in one concentrated place like multiple tables instead of each test.

Reply:
Okay, but I thought it would be useful too to have different names this way. If DB name and table name are the same for all tests (or repeated), then I feel there is lost opportunity to test new patterns